index.js

```diff
@@ -20,6 +20,10 @@ module.exports = function transformJSONToTable(docs, options = {}) {
         if (options.excludeSubArrays) {
           this.update(value, true);
         }
+        if (options.listSubArrays && !_.isPlainObject(value[0])) {
```

Comment:
I've not dived into this code personally, but if I understand this, it is only checking the first element in value - which value is a value in a potential "cell" of the table - to see if it meets the criteria of "a subarray that does not contain objects", right? 

It's probably a good enough check to just check the first element of value, but just want to be sure that's what we want and what users expect when using this feature. 

Also, it's possible there could be an empty array as a value, right? We may want to check value.length before accessing `value[0]`. Again, not sure the full context of this code, so I could be wrong.

Reply:
Thanks for the feedback! I will do some testing to make sure an empty array won't break anything. It's not currently breaking this code, but I'll check to see how it renders with the table in the frontend. 
As far as what the user is expecting, I'm trying to add an option for the dev (not the user) to list out values in a cell instead of breaking them up into individual cells/columns. I believe it was a request from a HubSpot user. So as far as I understand, this would accomplish that.

Reply:
@kelvcutler so I tested it out with empty arrays. If the data is uniform, it doesn't break anything code wise or rendering in the frontend. I did find odd behavior though for data that isn't uniform when it comes to arrays/objects. But it was from the existing codebase. I don't know if you want me to look into that or just leave it as is.